### DISCUSSION TITLE
follow unit test guide, get 'mockResolvedValue' does not exist on type error

### DISCUSSION THREAD

**Discussion by lzuliuyun** [prisma/prisma#8569] [Q&A] [answered]
* follow unit test guide: https://www.prisma.io/docs/guides/testing/unit-testing
* step by step use Singleton
* get the follow error
![image](https://user-images.githubusercontent.com/6896562/128119372-6b9c48dc-65ce-46f2-85d1-a2ab04013b6a.png)

**Comment by molebox** [comment, marked as answer]
Hey @lzuliuyun would you be able to share a reproduction repo please, would help in debugging why you're seeing this error. Thanks 😄

**Reply by molebox** [reply]
Hey @lzuliuyun, this was due to a breaking change introduced in the `jest-mock-extended` package. I've amended our docs with the fix [here](https://www.prisma.io/docs/guides/testing/unit-testing#mocking-the-prisma-client)

**Reply by PracticalMagicProgramming** [reply]
Hello I'm running into the same issue despite using the directions on the amended version of the documentation. Any guidance?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
